### DISCUSSION TITLE
How to get this to work on Ubuntu Gnome Desktop only instead of other Apps?

### DISCUSSION THREAD

**Discussion by macOS26** [AntiMicroX/antimicrox#855] [General]
Works, great, but it interferes with Wine64 and BigPEmu.exe.  I want to use it as an Application Launcher using the Game Pad. Which I have working but I don't know what the Desktop app is called or how to define it. 

I heard something about desktop only mode but I don't see what anywhere. I tried super user for wine64 but that doesn't work.

**Comment by macOS26** [comment]
basically want it to work on Workspace 0 or Workspace 1

**Comment by Qzhor** [comment]
me too, only works in desktop or input area

**Reply by macOS26** [reply]
Works great on the gnome desktop but its has deadzone issues on boards that have noise/feedback. No dead zone cancellation. 

Also when leaving an aoo does not automatically go back to default
